tests/e2e/hcp_cluster_test.go

```diff
@@ -657,4 +658,34 @@ var _ = Describe("HCP cluster testing",
 					}
 				}
 			})
+
+		It("create zero-egress HCP cluster - [id:76543]",
+			labels.High, labels.Runtime.Day1,
```

Comment:
I think it can move the step:CreateMachinePool in day2 case

Reply:
Even though its part of the test case? I know that having a machinepool created on a ZE cluster was an area we wanted tested.

Reply:
According to the rule, we have day1\day2\destoy ... cases. We will filter the case by orde.So if the case is for day1, it would be better to not contain day2 step. We can move it in day2 case other than ignore the step.

Reply:
No problem, I'll remove the mp portion.